PhotoKingdomAPI/PhotoKingdomAPI/Controllers/Manager.cs

```diff
@@ -408,6 +485,51 @@ public int loadData()
                         IsWinner = 0 // TODO: Remove
                     });
 
+                    var upload7 = ds.AttractionPhotowarUploads.Add(new AttractionPhotowarUpload
+                    {
+                        Photo = boyerwoodlotPhoto1,
+                        AttractionPhotoWar = photowar3,
+                        IsWinner = 1

```

Comment:
See my note above. Instead of hardcoding the winner here, you can put expiry date on the photowars, and then when we run api/CheckOwns, it  will determine the winner for each photowar based on the most votes. It also puts IsLoser = 0 for the winner, and IsWinner = 0, IsLoser = 1 for the loser, which may be needed on the front end, otherwise if it's null it may throw a nullexception in some of my code that check if IsWinner == 1.

Reply:
sameapplies for the other uploads below